src/modules/dataFetching/components/DeleteMutationButton.tsx

```diff
@@ -70,25 +71,30 @@ const DeleteMutationButton = <Mutation, MutationVariables>({
       <ConfirmationDialog
         id='deleteRecordDialog'
         open={showDialog}
-        title={`Delete ${recordName}`}
-        confirmText={`Yes, delete ${recordName}`}
+        title={`${verb} ${recordName}`}
+        confirmText={`Yes, ${verb} ${recordName}`}
+        cancelText='Close'
         onConfirm={handleDelete}
         onCancel={() => setShowDialog(false)}
         loading={loading}
         errorState={{ apolloError: error, errors: [], warnings: [] }}
         color='error'
         {...ConfirmationDialogProps}
       >
-        {confirmationDialogContent ? (
-          confirmationDialogContent
-        ) : (
-          <>
-            <Typography>
-              {`Are you sure you want to delete this ${recordName}?`}
-            </Typography>
-            <Typography>This action cannot be undone.</Typography>
-          </>
-        )}
+        <>
+          {error && <ApolloErrorAlert error={error} />}
```

Comment:
This isn't necessary because the `ConfirmationDialog` renders any errors passed in to `errorState`

Reply:
Ah oops, I somehow missed it was already handled- thank you!